lib/mix/lib/mix/task.ex

```diff
@@ -522,12 +604,26 @@ defmodule Mix.Task do
   end
 
   defp recur(fun) do
+    run_in_children_projects(fun, Mix.Dep.Umbrella.cached())
+  end
+
+  defp recur_in_apps(fun, apps) do
+    selected_children =
+      Mix.Dep.Umbrella.cached()
+      |> Enum.filter(fn %Mix.Dep{app: app} ->
+        app in apps
```

Comment:
Should we warn if one of the apps do not exist?

Reply:
Checks live now in `Mix.Task.Do`